### PR TITLE
Documented tokenManager "added" event

### PR DESCRIPTION
**Pull request by garrettmac**
Added missing docs

### CODE REVIEW

**Comment by robertjd** [comment]
Thanks @garrettmac , I've asked @aarongranick-okta to review this one, we'll get back to you.

OKTA-383337

**Review by aarongranick-okta — APPROVED** [review]
thanks for catching this! we will get this merged in

File: README.md [hunk 1]
```diff
@@ -2466,6 +2466,7 @@ Subscribe to an event published by the `tokenManager`.
 * `event` - Event to subscribe to. Possible events are:
   * `expired` - Fired before a token is set to expire (using `expireEarlySeconds` option, 30 seconds by default). If `autoRenew` option is set to true, a listener will be attached to this event and an attempt will be made to renew the token when the event fires.
   * `error` - Fired when a token renew attempt has failed. This is a permanent error, and the token will be removed from storage.
+  * `added` - Fired when a new token has been added or a new token has been added that did not change the original.
```
Comment (garrettmac) [inline README.md:2469]:
```suggestion
* `added` - Fired when a new token has been added or a token has been updated that did not change from the original value.
```

File: README.md [hunk 2]
```diff
@@ -2466,7 +2466,7 @@ Subscribe to an event published by the `tokenManager`.
 * `event` - Event to subscribe to. Possible events are:
   * `expired` - Fired before a token is set to expire (using `expireEarlySeconds` option, 30 seconds by default). If `autoRenew` option is set to true, a listener will be attached to this event and an attempt will be made to renew the token when the event fires.
   * `error` - Fired when a token renew attempt has failed. This is a permanent error, and the token will be removed from storage.
-  * `added` - Fired when a new token has been added or a new token has been added that did not change the original.
+* `added` - Fired when a new token has been added or a token has been updated that did not change from the original value.
```
Comment (aarongranick-okta) [inline README.md:2469]: Does this mean it fires when a token is added or updated, **whether or not** the update has changed the token value?

Comment (garrettmac) [reply README.md:2469]: im going off your unit tests 
https://github.com/okta/okta-auth-js/blob/dd664ebea67692b1878ee05ffc2c2bc2699e00f0/test%2Fspec%2FtokenManager.js#L1742

https://github.com/okta/okta-auth-js/blob/dd664ebea67692b1878ee05ffc2c2bc2699e00f0/test%2Fspec%2FtokenManager.js#L1759
looks like thats kinda what you're saying yeah?

Any suggestions on rewording?
open to a 
```suggestion suggestion...

That sound about right tho?

**Comment by shuowu** [comment]
Merged in #684

**Pull request closed without merging**